exercises/change/change_test.go

```diff
@@ -0,0 +1,68 @@
+package change
+
+import (
+	"reflect"
+	"testing"
+)
+
+const targetTestVersion = 1
+
+func TestTestVersion(t *testing.T) {
+	if testVersion != targetTestVersion {
+		t.Fatalf("Found testVersion = %v, want %v", testVersion, targetTestVersion)
+	}
+}
+
+func determineExpected(expected interface{}) (valid bool, list []int) {
```

Comment:
I personally was thinking this might not be 100% clear to the students. Any way this logic can be moved to the *generator* instead? That ties in with the above question about the `testCase` containing the `[]int` and `bool`.